### PR TITLE
Create 2015-06-17-box-gallery-NBA

### PR DESCRIPTION
**Pull request by aneda**
@chriddyp is this good for box gallery?  Also, I didn't know what to put as the name

### CODE REVIEW

File: _posts/python/box/2015-06-17-box-gallery-NBA [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            ),    
+            line = Line(
+                    width=1,    
+            ),
+            
+        ))
+
+
+layout = Layout(
+    title = 'Points Scored by the Top 50 Scoring NBA Players in 2012',
```
Comment (chriddyp) [inline _posts/python/box/2015-06-17-box-gallery-NBA]: change to "Top 9"

File: _posts/python/box/2015-06-17-box-gallery-NBA [hunk 2]
```diff
@@ -0,0 +1,78 @@
+---
+name: Box Plot Distribution Comparison between Several Groups
```
Comment (chriddyp) [inline _posts/python/box/2015-06-17-box-gallery-NBA]: Maybe just "Fully Styled Box Plots"

**Comment by aneda** [comment]
@chriddyp is it good?

**Comment by chriddyp** [comment]
:dancer:

**Pull request merged**